src/Prefetch.h

```diff
@@ -9,11 +9,15 @@
 #include <map>
 
 #include "IR.h"
+#include "Schedule.h"
 #include "Target.h"
 
 namespace Halide {
 namespace Internal {
 
+Stmt inject_def_empty_prefetch(Stmt s, const std::map<std::string, Function> &env,
```

Comment:
What does 'def' mean here? Consider renaming to inject_placeholder_prefetch?

Also, add comment on this.

Reply:
This method injects placeholder prefetch per Function Definition. I was trying to make it explicit that this should be called per Definition during ScheduleFunctions